modules/hello-world/examples/IndexHelloWorld.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2021 Couchbase, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import java.time.Duration;
+import java.util.Arrays;
+import com.couchbase.client.java.Cluster;
+import com.couchbase.client.java.manager.query.CreatePrimaryQueryIndexOptions;
+import com.couchbase.client.java.manager.query.CreateQueryIndexOptions;
+import com.couchbase.client.java.manager.query.WatchQueryIndexesOptions;
+
+public class IndexHelloWorld {
+
+  public static void main(String... args) {
+    Cluster cluster = Cluster.connect("localhost", "Administrator", "password");
+
+    {
+      System.out.println("[primary]");
+      // tag::primary[]
```

Comment:
I think the Java SDK lets you create and drop named primary indexes? (with the options object) Again, it's not strictly necessary, but it would be good to have examples of this also.

Reply:
You can create but not drop for some reason form what I can see:
https://docs.couchbase.com/sdk-api/couchbase-java-client/com/couchbase/client/java/manager/query/DropPrimaryQueryIndexOptions.html#dropPrimaryQueryIndexOptions()

Reply:
Yes of course, we already discussed this, my bad.